src/main/kotlin/com/launchdarkly/intellij/settings/LaunchDarklyApplicationSettings.kt

```diff
@@ -130,46 +128,69 @@ class LaunchDarklyApplicationConfigurable : BoundConfigurable(displayName = "Lau
 
         panel = panel {
             row {
-                comment("Add your LaunchDarkly API Key and click Get Projects. Project and Environment selections will populate based on key permissions.")
+                text("Access feature flags in the IDE without having to navigate away from your current workflow.")
             }
-            row("API Key:") {
-                cell(apiField).bindText(settings::authorization).columns(COLUMNS_MEDIUM)
-                button("Get Projects") {
-                    updateProjects(
-                        panel as DialogPanel,
-                        it
-                    )
-                }
+
+            row {
+                cell(accessTokenField)
+                    .label("Access token:")
+                    .bindText(settings::authorization)
+                    .columns(COLUMNS_MEDIUM)
+                    .validationOnInput(apiKeyValidation())
+                icon(AllIcons.General.BalloonWarning)
+                    .label("Apply changes")
+                    .visibleIf(refreshProjectsPredicate())
+            }
+
+            row {
+                comment("Input the access token from your LaunchDarkly account. If you don’t have one, you must <a href=\"https://docs.launchdarkly.com/home/account-security/api-access-tokens#creating-api-access-tokens\">create an access token</a> first.")
             }
 
             try {
                 projectBox = if (::projectContainer.isInitialized) {
                     DefaultComboBoxModel(projectContainer.map { it.key }.toTypedArray())
                 } else {
-                    DefaultComboBoxModel(arrayOf(defaultMessage))
-                }
-                row("Project") {
-                    comboBox(projectBox, renderer).bindItem(settings::project)
+                    DefaultComboBoxModel()
                 }
 
                 environmentBox = if (::environmentContainer.isInitialized) {
                     DefaultComboBoxModel(environmentContainer.environments.map { it.key }.toTypedArray())
                 } else {
-                    DefaultComboBoxModel(arrayOf("Please select a Project"))
-                }
-                row("Environments:") {
-                    comboBox(environmentBox, renderer).bindItem(settings::environment)
+                    DefaultComboBoxModel()
                 }
                 environmentBox.selectedItem = settings.environment
+
+                indent {
```

Comment:
will remove in future ticket https://app.shortcut.com/launchdarkly/story/145970/investigate-and-resolve-plugin-verification-issues